lib/ex_unit/test/ex_unit/doc_test_test.exs

```diff
@@ -206,64 +215,65 @@ defmodule ExUnit.DocTestTest do
 
     ExUnit.configure(seed: 0, colors: [enabled: false])
     output = capture_io(fn -> ExUnit.run end)
+             |> String.replace(~r/exs:\d+/, "exs:x") # Ignore line numbers
```

Comment:
The thing is that the line numbers are important to test too. I know it is a pain because when we update the tests, we need to update the line numbers, but if we don't test them here, we won't test them anywhere. So we should rather revert this change. :)